[2022-08-02 13:03:16,507] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Demo.data_uploading manual__2022-08-02T13:03:08.564821+00:00 [queued]>
[2022-08-02 13:03:16,518] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Demo.data_uploading manual__2022-08-02T13:03:08.564821+00:00 [queued]>
[2022-08-02 13:03:16,519] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-02 13:03:16,520] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-02 13:03:16,520] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-02 13:03:16,537] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): data_uploading> on 2022-08-02 13:03:08.564821+00:00
[2022-08-02 13:03:16,542] {standard_task_runner.py:52} INFO - Started process 10572 to run task
[2022-08-02 13:03:16,545] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Demo', 'data_uploading', 'manual__2022-08-02T13:03:08.564821+00:00', '--job-id', '294', '--raw', '--subdir', 'DAGS_FOLDER/demo.py', '--cfg-path', '/tmp/tmp028t2990', '--error-file', '/tmp/tmpb2u9djhk']
[2022-08-02 13:03:16,546] {standard_task_runner.py:80} INFO - Job 294: Subtask data_uploading
[2022-08-02 13:03:16,624] {task_command.py:371} INFO - Running <TaskInstance: Demo.data_uploading manual__2022-08-02T13:03:08.564821+00:00 [running]> on host 31353b71d6cb
[2022-08-02 13:03:16,713] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Demo
AIRFLOW_CTX_TASK_ID=data_uploading
AIRFLOW_CTX_EXECUTION_DATE=2022-08-02T13:03:08.564821+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-02T13:03:08.564821+00:00
[2022-08-02 13:03:16,714] {python.py:173} INFO - Done. Returned value was: 0
[2022-08-02 13:03:16,744] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=Demo, task_id=data_uploading, execution_date=20220802T130308, start_date=20220802T130316, end_date=20220802T130316
[2022-08-02 13:03:16,797] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-02 13:03:16,854] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
